sdk/metric/view/view_test.go

```diff
@@ -0,0 +1,348 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package view
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/sdk/instrumentation"
+)
+
+var matchInstrument = Instrument{
+	Scope: instrumentation.Library{
+		Name:      "bar",
+		Version:   "v1.0.0",
+		SchemaURL: "stuff.test/",
+	},
+	InstrumentName:        "foo",
+	InstrumentDescription: "",
+}
+var noMatchInstrument = Instrument{
+	Scope: instrumentation.Library{
+		Name:      "notfoo",
+		Version:   "v0.x.0",
+		SchemaURL: "notstuff.test/",
+	},
+	InstrumentName:        "notstuff",
+	InstrumentDescription: "",
+}
+
+var emptyDescription = Instrument{}
+
+func TestConfig_Transform(t *testing.T) {
```

Comment:
This test both matching and transforms. Can you split the match testing to its own test function?

Reply:
This is a test of all the many things that view.TransformInstrument does.  We can have a separate test that just is the matching, but the test for transformation needs to also have matching.  Both because it's an error to not include any match, and to make sure that not matching doesn't transform an empty to something else.